packages/react-ux-capture-example/src/Bar.jsx

```diff
@@ -1,14 +1,24 @@
 import React, { Component } from 'react';
-import Inline from './marks/Inline';
-import Lazy from './Lazy';
 
+import UXCaptureInlineMark from '@meetup/react-ux-capture/lib/UXCaptureInlineMark';
```

Comment:
A general comment for a few of these files - 

It'd be great to have more descriptive/user-friendly names instead of 'Foo.jsx', 'Bar.jsx', etc.

I suspect that both the demo and the source code is going to be used as a reference implementation, so making it more descriptive would be helpful.